playbooks/ceph.yml

```diff
@@ -163,6 +124,15 @@
         ms_bind_ipv4: false
         ms_bind_ipv6: true
 
+    - name: Debug ssh_network_range
+      ansible.builtin.debug:
+        var: ssh_network_range
+
+    - name: Debug all_addresses
+      ansible.builtin.debug:
+        msg: "{{ hostvars[item][all_addresses] }}"
+      loop: "{{ groups[cifmw_ceph_target | default('computes')] | default([]) }}"
```

Comment:
```
2024-08-22 14:59:58,512 p=29984 u=zuul n=ansible | TASK [Debug ssh_network_range var=ssh_network_range] ***************************
2024-08-22 14:59:58,512 p=29984 u=zuul n=ansible | Thursday 22 August 2024  14:59:58 -0400 (0:00:00.033)       0:27:28.730 ******* 
2024-08-22 14:59:58,531 p=29984 u=zuul n=ansible | ok: [localhost] => 
  ssh_network_range: 192.168.122.0/24
2024-08-22 14:59:58,538 p=29984 u=zuul n=ansible | TASK [Debug all_addresses msg={{ hostvars[item][all_addresses] }}] *************
2024-08-22 14:59:58,538 p=29984 u=zuul n=ansible | Thursday 22 August 2024  14:59:58 -0400 (0:00:00.025)       0:27:28.756 ******* 
2024-08-22 14:59:58,564 p=29984 u=zuul n=ansible | fatal: [localhost]: FAILED! => 
  msg: |-
    The task includes an option with an undefined variable. The error was: 'ansible.vars.hostvars.HostVarsVars object' has no attribute 'ansible_all_ipv4_addresses'. 'ansible.vars.hostvars.HostVarsVars object' has no attribute 'ansible_all_ipv4_addresses'
  
    The error appears to be in '/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/playbooks/ceph.yml': line 131, column 7, but may
    be elsewhere in the file depending on the exact syntax problem.
  
    The offending line appears to be:
  
  
        - name: Debug all_addresses
          ^ here
```

https://logserver.rdoproject.org/50/2250/71192a64fba6152631de7cd267de29e9f7c5771b/github-check/podified-multinode-hci-deployment-crc/8f10dfd/controller/ci-framework-data/logs/ansible.log

Reply:
The above is from `podified-multinode-hci-deployment-crc`.

In my env when I ran with this patch I saw the following. Odd that I don't see the storage network. Regardless I'll be restoring the old block device role for `podified-multinode-hci-deployment-crc`

```
TASK [Debug ssh_network_range var=ssh_network_range] ***************************
Thursday 22 August 2024  15:43:25 -0400 (0:00:00.024)       0:00:06.625 ******* 
ok: [localhost] => 
  ssh_network_range: 192.168.122.0/24

TASK [Debug all_addresses msg={{ hostvars[item][all_addresses] }}] *************
Thursday 22 August 2024  15:43:26 -0400 (0:00:00.026)       0:00:06.652 ******* 
ok: [localhost] => (item=compute-0) => 
  msg:
  - 192.168.111.100
  - 192.168.122.100
ok: [localhost] => (item=compute-1) => 
  msg:
  - 192.168.122.101
  - 192.168.111.101
ok: [localhost] => (item=compute-2) => 
  msg:
  - 192.168.111.102
  - 192.168.122.102

TASK [Build a dict mapping hostname to its IP which is in management network range host_to_ip={{ host_to_ip | default({}) | combine( { item : hostvars[item][all_addresses] | ansible.utils.ipaddr(ssh_network_range) | first } ) }}] ***
Thursday 22 August 2024  15:43:26 -0400 (0:00:00.056)       0:00:06.708 ******* 
ok: [localhost -> compute-0(compute-0.utility)] => (item=compute-0)
ok: [localhost -> compute-1(compute-1.utility)] => (item=compute-1)
ok: [localhost -> compute-2(compute-2.utility)] => (item=compute-2)

TASK [Debug host_to_ip var=host_to_ip] *****************************************
Thursday 22 August 2024  15:43:26 -0400 (0:00:00.135)       0:00:06.843 ******* 
ok: [localhost] => 
  host_to_ip:
    compute-0: 192.168.122.100
    compute-1: 192.168.122.101
    compute-2: 192.168.122.102
```
and later...
```
[ceph: root@compute-0 /]# ceph -s
  cluster:
    id:     5ba40468-1ab3-5d13-bf47-9702459ceb10
    health: HEALTH_WARN
            clock skew detected on mon.compute-2, mon.compute-1
 
  services:
    mon: 3 daemons, quorum compute-0,compute-2,compute-1 (age 17h)
    mgr: compute-0.cnegzp(active, since 17h), standbys: compute-2.yihmho, compute-1.xznxfu
    mds: 1/1 daemons up, 2 standby
    osd: 9 osds: 9 up (since 17h), 9 in (since 17h)
    rgw: 3 daemons active (3 hosts, 1 zones)
 
  data:
    volumes: 1/1 healthy
    pools:   11 pools, 1297 pgs
    objects: 258 objects, 41 MiB
    usage:   713 MiB used, 269 GiB / 270 GiB avail
    pgs:     1297 active+clean
 
[ceph: root@compute-0 /]# 
```